dev-docs/bidders/freedomadnetwork.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: bidder
+title: Freedom Ad Network
+description: Freedom Ad Network Bidder Adapter
+biddercode: freedomadnetwork
+aliasCode: fan
```

Comment:
```suggestion
```

Sorry for not noticing before. Your adapter doesn't specify this as an alias, so this should be removed.

Reply:
Oh, thanks for noticing! I thought the alias had to be ${alias}Adapter.js in the format of the filename.